app/views/content/life-as-a-teacher/pay-and-benefits/bens-favourite-things-about-teaching/_article.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+      <p>What I particularly love about my school is the excitement and energy I can feel in the corridors. My students are enthusiastic and they push me to understand more about physics and maths too. It also helps that my colleagues are really friendly and supportive. I know that everyone is pulling in the same direction, and we have a good joke in the office.</p>
+      <p>A few years after I finished university, my old school sent out a letter to former students advertising their teacher training course. Around the same time, I went to meet one of my friends who was working at a Get Into Teaching event and got chatting about a career in teaching. It seemed like a great opportunity!</p>
+  </section>
+  <section class="col col-720">
+    <%= render Content::QuoteComponent.new(
+    text: "I like that I'm able to bring myself into the classroom.",
```

Comment:
Should this be 'bring my whole-self'? Sounds kinda weird as he will be in the classroom, otherwise he wouldn't be able to teach? Might be overthinking it!!